src/legacy/core_plugins/vis_type_markdown/public/markdown_vis_controller.tsx

```diff
@@ -17,14 +17,20 @@
  * under the License.
  */
 
-import React, { Component } from 'react';
+import React from 'react';
 import { Markdown } from 'ui/markdown/markdown';
 
+interface MarkdownVisComponentProps {
+  renderComplete: () => {};
+  markdown: string;
+  fontSize: number;
+  openLinksInNewTab: boolean;
+}
```

Comment:
I'm wondering if, instead of retyping the vis params here, we should instead be importing the `Arguments` interface from `markdown_fn`. That way it's easier to know that your component will break if you make changes to the expression function:

Maybe something like:
```typescript
import { MarkdownVisArguments } from './markdown_fn';

interface MarkdownVisComponentProps extends MarkdownVisArguments {
  renderComplete: () => {};
}
```

Or I guess the "right" approach would be to import the parts of the `Return` value that you need -- since _technically_ that's what this component is receiving -- but that's primarily just semantics and would require more TypeScript hackery to get at the values you need.